frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/volume-snapshot-modal.tsx

```diff
@@ -41,16 +41,18 @@ export const VolumeSnapshotModal = withHandlePromise((props: VolumeSnapshotModal
   const [snapshotCount, setSnapshotCount] = React.useState(3); // For all schedules
   const [snapshotWeek, setSnapshotWeek] = React.useState(0); // For Weekly schedule
   const [snapshotTime, setSnapshotTime] = React.useState('00:01'); // For weekly and monthly
-  const [snapshotMonth, setSnapshotMonth] = React.useState(1); // For monthly schedule
-  const [snapshotName, setSnapshotName] = React.useState<string>();
+  const [snapshotMonth, setSnapshotMonth] = React.useState(0); // For monthly schedule
```

Comment:
This is named as `snapshotMonth` but I see only month day is being set.

Reply:
ack